modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -209,6 +209,11 @@ Events:     <11>
 <10> The pods on the node.
 <11> The events reported by the node.
 
+[NOTE]
+====
+The control plane label is not automatically added to newly created or updated master nodes. If you want to use the control plane label for your nodes, you can manually configure the label. For more information, see _Understanding how to update labels on nodes_ in the _Additional resources_ section.
+====
+
```

Comment:
We should hide this in ROSA/OSD with ifndef's, because the _Additional resources_ section is hidden in ROSA/OSD.

Reply:
That makes sense! Thanks!